modules/nf-core/skani/triangle/main.nf

```diff
@@ -0,0 +1,49 @@
+process SKANI_TRIANGLE {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/skani:0.2.1--h4ac6f70_0':
+        'biocontainers/skani:0.2.1--h4ac6f70_0' }"
+
+    input:
+    tuple val(meta) , path(queries)
```

Comment:
`-l` as before, 

quries are FASTA files right? Might be clearer to just call it that.